index.html

```diff
@@ -1265,6 +1317,12 @@
       <label for="globCols"><span class="check"></span>global coloring</label>
     </div>
   </fieldset>
+  <fieldset>
+    <legend>Save SVG</legend>
```

Comment:
A global "_Save_" section would be great, with maybe those options ?

- "*Save*" (< would trigger a browser _save as_)
- "*Export SVG*"
- "*Print / Export PDF*" (< would trigger `print()` )

Reply:
👍